src/main/java/io/zeebe/Application.java

```diff
@@ -9,17 +9,45 @@
 import java.util.Map;
 import java.util.Scanner;
 
+/**
+ * Sample application that connects to a cluster on Camunda Cloud, or a locally deployed cluster.
+ *
+ * <p>When connecting to a cluster in Camunda Cloud, this application assumes that the following
+ * environment variables are set:
+ *
+ * <ul>
+ *   <li>ZEEBE_ADDRESS
+ *   <li>ZEEBE_CLIENT_ID
+ *   <li>ZEEBE_CLIENT_SECRET
+ *   <li>ZEEBE_AUTHORIZATION_SERVER_URL
+ * </ul>
+ *
+ * <p><strong>Hint:</strong> When you create client credentials in Camunda Cloud you have the option
+ * to download a file with above lines filled out for you.
+ *
+ * <p>When connecting to a local cluster, you can specifiy that address either at the command line
+ * or by setting {@code ZEEBE_ADDRESS}. This application also assumes that auhentication is disabled
+ * for a locally deployed cluster
+ */
 public class Application {
 
   public static void main(String[] args) {
-    final String contactPoint = args.length >= 1 ? args[0] : "127.0.0.1:26500";
+    final String defaultAddress = "127.0.0.1:26500";
+    final String addressFromCommandLine = args.length >= 1 ? args[0] : null;
+    final String addressFromEnvironmentVariable = System.getenv("ZEEBE_ADDRESS");
```

Comment:
Is this the only env var that is read? Shouldn't it also read ZEEBE_CLIENT_ID, etc?

Reply:
It does read that deep within the Zeebe client code.

Reply:
This is the only one (to the best of my knowledge) that needs to be passed in. Don't ask me why.

Reply:
If that happens within the zeebe-java-client code, then I'm not sure about this sentence: https://github.com/zeebe-io/zeebe-get-started-java-client/pull/6/files#diff-f3bc9309de1cd79fc2729102d2967aa5b7f12c41a2c3413188c2a9255b4a9631R15. Perhaps it can say what this application explicitly requires, and what the zeebe-java-client can do

Reply:
I see what you mean. I just don't know how to phrase it without confusing the reader. (I am myself confused and not a fan of code reading environment variables in arbitrary places)

Do you have a suggestion how to phrase this?

Reply:
What about:
```
 * <ul>
 *   <li>ZEEBE_ADDRESS
 *   <li>ZEEBE_CLIENT_ID (implicitly required by zeebe-java-client)
 *   <li>ZEEBE_CLIENT_SECRET (implicitly required by zeebe-java-client)
 *   <li>ZEEBE_AUTHORIZATION_SERVER_URL (implicitly required by zeebe-java-client)
 * </ul>
```

or 
```
 * <ul>
 *   <li>ZEEBE_ADDRESS (specific to this application, the others are read by the zeebe-java-client)
 *   <li>ZEEBE_CLIENT_ID
 *   <li>ZEEBE_CLIENT_SECRET
 *   <li>ZEEBE_AUTHORIZATION_SERVER_URL
 * </ul>
```